harvester/network-controller-harvester
# Bump go mod dependency

**Problem:**
Go mod contains some outdated dependencies.

**Solution:**
Bump go mod dependency and fix some library issues

**Related Issue:**
https://github.com/harvester/security/issues/29


**Test plan:**
- build the image
- verify the trivy scanning result
- verify the `network-controller` is up and ready; it should not contain any error logs.
